scenes/act4.md

```diff
@@ -428,86 +428,2006 @@ h: Okay. Healthy relationship with emotions. Relationships need communication. S
+
+`bb({ eyes:"normal" });`
+
+h: Vero, non c'é modo di proteggerci al 100%...
+
+`hong({ body:"one_up" });`
+
+h: Ma anche un miglioramento dell' 1% vale la pena, giusto?
+
+```
+bb({ eyes:"annoyed" });
+hong({ normal:"one_up" });
+```
+
+b: Vedi il bicchiere non come vuoto al 99%, ma pieno all' 1%?
+
+`bb({ eyes:"normal" });`
+
+h: Che é ancora un bene prezioso, se sei bloccato in mezzo al deserto.
+
+`bb({ eyes:"closed" });`
+
+b: Beh. Alla goccia, allora.
+
+(#act4_something_else)
+
+# act4_harm_skills_overboard
+
+`bb({ body:"chest", eyes:"annoyed" })`
+
+b: I mean, the whole reason you ignored my warnings was because *I* went overboard with safety! 
+b: Voglio dire, la ragione per cui hai ignorato i miei avvertimenti é perché ho esagerato con la sicurezza! 
+
+`bb({ body:"normal", eyes:"normal" })`
+
+h: Nah, hai ragione. Vorremmo pensare alla sicurezza con moderazione. Tutto con moderazione.
+
+`bb({ eyes:"suspect" })`
+
+b: Scusa, *TUTTO* con moderazione?
+
+`hong({ eyes:"annoyed" })`
+
+h: *Un numero moderato di cose* con moderazione.
+
+```
+bb({ eyes:"closed" });
+hong({ eyes:"normal" });
+```
+
+b: Grazie per aver reso le tue dichiarazioni ricorsivamente autoconsistenti.
+
+(#act4_something_else)
+
+
+# act4_harm_exposure
+
+`bb({ mouth:"scream_talk", eyes:"scream", MOUTH_LOCK:true });`
+
+b: *COSA*
+
+```
+bb({ mouth:"narrow", eyes:"suspect" });
+hong({ body:"one_up" });
+```
+
+h: Voglio dire, diciamo che un cane ha paura dei tuoni.
+
+`hong({ body:"hands_1" });`
+
+h: Un trucco usato dagli addestrattori é fargli ascoltare un tuono a basso volume mentre gli danno dei premi per tenerli calmi.
+
+`hong({ body:"hands_2" });`
+
+h: Nei giorni successivi, l'addestratore inizia ad alzare il volume, fino a quando il cane non ha più paura.
+
+```
+hong({ body:"normal", eyes:"surprise" });
+bb({ mouth:"normal", eyes:"normal" });
+```
+
+h: Si chiama terapia espositiva!
+
+`hong({ body:"point", eyes:"normal" });`
+
+h: Dato che sei un cane dovrebbe funzionare, giusto? Tutti i mammiferi hanno lo stesso istinto di sopravvivenza.
+
+`hong({ body:"normal" });`
+
+[E se ci desensibilizziamo *troppo*?](#act4_harm_exposure_overboard)
+
+[E se ci esponiamo a del *vero* pericolo?](#act4_harm_exposure_hurt)
+
+[Sono un lupo, non un cane.](#act4_harm_exposure_dog) `bb({ eyes:"suspect" })`
+
+# act4_harm_exposure_dog
+
+h: E ti mostrero gentilezza e pazienza fino ad addomesticarti, e farti sembrare un cucciolo.
+
+`bb({ MOUTH_LOCK:true })`
+
+b: ...
+
+`bb({ eyes:"sad", mouth:"smile" })`
+
+b: D'aw.
+
+(#act4_something_else)
+
+# act4_harm_exposure_overboard
+
+`bb({ eyes:"annoyed" })`
+
+b: Abbiamo appena visto cosa succede quando ignori le tue paure, metti te stesso in *serio* pericolo.
+
+`bb({ eyes:"angry_r", body:"one_up" })`
+
+b: Comunque, *troppa* desensibilizzazione non ci fará diventare psychopatici?
+
+`bb({ mouth:"scream", eyes:"scream", body:"two_up" })`
+
+b: Soon we'll give ourselves treats while watching snuff murder porn!
```

Comment:
presto ci daremo piacere guardando pornografia sodomitica!

Reply:
da una ricerca su internet pero' lo snuff murder porn non e' proprio quello - e' vedere gente che viene uccisa in diretta/in video e torturata per davvero, non a fini di intrattenimento porno. Alla fine il BDSM consensuale e' un fetish sano, ma qui parlano di diventare psicopatici e dei serial killer (piu' in la' nella traduzione). non vorrei mettere il BDSM consensuale in quella categoria.